frontend/app/globals.css

```
@@ -0,0 +1,95 @@
+}
+
+html {
+  scroll-behavior: smooth;
+  font-size: 112.5%; /* 18px 기준 — 전체 텍스트 가독성 상향 */
+}
+
+body {
+  background: var(--bg);
+  color: var(--text);
+  font-family:
+    "Pretendard",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)